### PR TITLE
Added new AB video test

### PR DESCRIPTION
**Pull request by natapg**
Closes WEB-112

### CODE REVIEW

File: src/components/YoutubeEmbed/YoutubeEmbed.tsx [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+  return (
+    <div className={styles.videoResponsive}>
+      <iframe
+        width='853'
+        height='480'
+        src={`https://www.youtube.com/embed/${embedId}`}
+        frameBorder='0'
+        allow='accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture'
+        allowFullScreen
+        title='Embedded youtube'
```
Comment (tan-90) [inline src/components/YoutubeEmbed/YoutubeEmbed.tsx:18]: The iframe should have a descriptive title passed as a prop.

File: src/components/LiveDemoSection/index.tsx [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+import YoutubeEmbed from '../YoutubeEmbed/YoutubeEmbed'
+
+import styles from './LiveDemoSection.module.scss'
+
+export default function LiveDemoSection() {
+  return (
+    <Section className={styles.liveDemo}>
+      <Container size='large' className={styles.container}>
+        <div className={styles.content}>
+          <h2 className={styles.title}>Installing FingerprintJS Pro</h2>
```
Comment (tan-90) [inline src/components/LiveDemoSection/index.tsx:14]: I don't love the word "Installing" in this situation. I'd go with something like "Getting started", but that's just a personal nitpick.

**Review by tan-90 — CHANGES_REQUESTED** [review]

File: src/components/LiveDemoSection/index.tsx (hunk 2 above)
Comment (natapg) [reply src/components/LiveDemoSection/index.tsx:14]: I agree, I'm going to ask Savannah

**Review by tan-90 — APPROVED** [review]

**Review by Lisenish — CHANGES_REQUESTED** [review]
@natapg Mostly minor comments, good job Nata!

File: src/components/YoutubeEmbed/YoutubeEmbed.module.scss [hunk 3]
```diff
@@ -0,0 +1,19 @@
+.videoResponsive {
+  overflow: hidden;
+  padding-bottom: 56.25%;
```
[the remark this answers is not in the document]
Comment (natapg) [reply src/components/YoutubeEmbed/YoutubeEmbed.module.scss:3]: a percentage value for padding-bottom and the position in relative keep the aspect ratio of the container ( ie 56.25% = 16:9 )

File: src/components/YoutubeEmbed/YoutubeEmbed.module.scss [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+    width: 100%;
+  }
+}
+
+.videoResponsive iframe {
+  left: 0;
+  top: 0;
+  height: 100%;
+  width: 100%;
+  position: absolute;
```
[the remark this answers is not in the document]
Comment (natapg) [reply src/components/YoutubeEmbed/YoutubeEmbed.module.scss:18]: Is to take a fixed position relative to the parent <div> and keep the aspect ratio.
Changing it to relative or static looks like this
image

File: src/components/LiveDemoSection/LiveDemoSection.module.scss [hunk 5]
```diff
@@ -1,62 +1,77 @@
...
-  flex-direction: column;
+.container {
+  display: grid;
+  grid-template-columns: repeat(2, 1fr);
   align-items: center;
+  @include gap(5rem);
 
-  background-repeat: no-repeat;
-  background-size: 1400px auto;
+  > * + * {
```
[the remark this answers is not in the document]
Comment (natapg) [reply src/components/LiveDemoSection/LiveDemoSection.module.scss:7]: Since it is not my code, it will take me a while to analyze it

**Comment by natapg** [comment]
@Lisenish done 👍

File: src/components/LiveDemoSection/index.tsx [hunk 6]
```diff
@@ -2,7 +2,8 @@ import React from 'react'
 import FpjsWidget from '../FpjsWidget'
 import Container from '../common/Container'
 import Section from '../common/Section'
-import YoutubeEmbed from '../YoutubeEmbed/YoutubeEmbed'
+import YoutubeEmbed from '../YoutubeVideo/YoutubeVideo'
```
Comment (Lisenish) [inline src/components/LiveDemoSection/index.tsx:5]: Rename this too please

**Review by Lisenish — APPROVED** [review]

**Comment by Lisenish** [comment]
Closing PR, since it will be our current AB test branch

**Pull request closed without merging**